interview-questions.md

```diff
@@ -8,50 +8,64 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is the difference between a parameter and an argument?
 
-Your answer:
+Your answer: The parameter is the place holder inside of the parenthesis. The argument is the actual content within the parameter
 
-Researched answer:
+Researched answer: A parameter is a place holder variable that is defined inside the parenthesis of the function expression. Once it is defined in the parenthesis, it can be leveeraged anywhere in the function logic.
+
+An argument is the actual content passed to the function inside the parenthesis of the function invocation. The argument value must be a data type recongized by javascript.
+
```

Comment:
I love that you are using the syllabus but researched answers need to be in your own words.